src/server/Adaptive.ReactiveTrader.Server.IntegrationTests/TestAddress.cs

```diff
@@ -2,6 +2,6 @@ namespace Adaptive.ReactiveTrader.Server.IntegrationTests
 {
     public static class TestAddress
     {
-        public const string Broker = "ws://broker:8000/ws";
+        public const string Broker = "ws://127.0.0.1:8000/ws";
```

Comment:
Did you mean to check this in? Will this work on a deployed environment?

Reply:
Yes. All the server  Integration tests both locally (and on the build server) are broken.
This was the original value by Qiming and I changed it to this to get the tests running locally.
It was changed to "broker" on 17 Jan by Keshai which broke the tests.

I want to see if this fixes the build server tests. If it does then running on deployed environments obviously needs more work.

Reply:
I have changed it back to "broker". "localhost" allows the tests to run locally in VS2019 if I have already started the Service.
Someone who knows the build server will have to fix this asap.